controllers/sightingsController.js

```diff
@@ -1,20 +1,141 @@
 const BaseController = require("./baseController");
 
 class SightingsController extends BaseController {
-  constructor(model) {
+  constructor(model, commentModel, categoryModel) {
     super(model);
+    this.commentModel = commentModel;
+    this.categoryModel = categoryModel;
   }
 
   // Retrieve specific sighting
   async getOne(req, res) {
     const { sightingId } = req.params;
     try {
-      const sighting = await this.model.findByPk(sightingId);
+      const sighting = await this.model.findByPk(sightingId, {
+        include: "categories",
+      });
       return res.json(sighting);
     } catch (err) {
       return res.status(400).json({ error: true, msg: err });
     }
   }
+
+  // create new sighting
+  async insertOne(req, res) {
+    const { date, location, notes, selectedCatIds } = req.body;
+    try {
+      const newSighting = await this.model.create({
+        date: date,
+        location: location,
+        notes: notes,
+      });
+
+      const selectedCategory = await this.categoryModel.findAll({
```

Comment:
```suggestion
      const selectedCategories = await this.categoryModel.findAll({
```

findAll returns a list, so should be plural